ui/articles-aem-renderer/src/androidTest/java/org/cru/godtools/articles/aem/db/ArticleDBTest.java

```diff
@@ -0,0 +1,145 @@
+package org.cru.godtools.articles.aem.db;
+
+import android.arch.lifecycle.Lifecycle;
+import android.arch.lifecycle.LifecycleOwner;
+import android.arch.lifecycle.Observer;
+import android.arch.persistence.room.Room;
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.cru.godtools.articles.aem.model.Article;
+import org.cru.godtools.articles.aem.model.Attachment;
+import org.cru.godtools.articles.aem.model.ManifestAssociation;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+
+import timber.log.Timber;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ArticleDBTest implements LifecycleOwner {
```

Comment:
Test classes should follow the naming scheme `{classBeingTested}Test`. So in this case this should be `ArticleRoomDatabaseTest`